### PR TITLE
fix: Docker with Conda & PT 1.8

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

try to fix the failing Horovod build for PT 1.8

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@tgaddair mind have look at the installation, there seems to be something wrong with parsing PT versions
when I list installed torched versions
```
$ pip list | grep torch
torch                     1.8.0.dev20210205
torchtext                 0.9.0.dev20210205
torchvision               0.9.0.dev20210205
```
but later in horovod installation, I get:
```
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
Could NOT find Pytorch: Found unsuitable version "", but required is at least "1.2.0" (found )
```

**Comment by Borda** [comment]
ok, the problem is rather in PT itself - https://github.com/pytorch/pytorch/issues/51026

File: dockers/README.md [hunk 1]
```diff
@@ -20,6 +20,29 @@ docker image build \
...
+```bash
+docker image build \
+    -t pytorch-lightning:py3.7-pt1.8 \
+    -f dockers/base-conda/Dockerfile \
+    --build-arg PYTHON_VERSION=3.7 \
+    --build-arg PYTORCH_VERSION=1.8 \
+    --build-arg PYTORCH_CHANNEL="pytorch-nightly" \
+    .
+```
+
```
Comment (Borda) [inline dockers/README.md:45]:
```suggestion
```

File: dockers/base-conda/Dockerfile [hunk 2]
```diff
@@ -43,6 +43,7 @@ RUN apt-get update -qq && \
         curl \
         unzip \
         ca-certificates \
+        libgomp1 \
```
Comment (Borda) [inline dockers/base-conda/Dockerfile:46]:
```suggestion
```

**Comment by Borda** [comment]
let's update conda too https://github.com/conda/conda/issues/8385#issuecomment-719466443

**Review by carmocca — APPROVED** [review]

**Review by s-rog — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by Borda** [comment]
Screenshot 2021-02-09 at 9 42 13

**Pull request merged**